src/openforms/js/components/admin/form_versions/FormVersionsTable.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import useAsync from 'react-use/esm/useAsync';
+
+import {get, apiCall} from '../../../utils/fetch';
+import {FORM_ENDPOINT} from '../form_design/constants';
+import Loader from "../Loader";
+
+
+const FormVersionsTable = ({ csrftoken, formUuid, formAdminUrl}) => {
+    const [formVersions, setFormVersions] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const restoreVersion = async (csrftoken, formUuid, versionUuid, redirectUrl) => {
+        await apiCall(
+            `${FORM_ENDPOINT}/${formUuid}/versions/${versionUuid}/restore`,
+            {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'X-CSRFToken': csrftoken,
+                },
+            }
+        );
+        window.location = redirectUrl;
+    };
+
+    const getFormVersions = async (uuid) => {
+        const response = await get(
+            `${FORM_ENDPOINT}/${uuid}/versions`
+        );
+        setFormVersions(response.data);
+        setIsLoading(false);
+    };
+
+    useAsync(async () => {await getFormVersions(formUuid);}, []);
+
+    const rows = formVersions.map((version, index) => {
+        const dateCreation = new Date(version.dateCreation);
+        return (
+           <tr key={index}>
+                <th>{ dateCreation.toDateString() } { dateCreation.toLocaleTimeString() }</th>
+                <td><a href="#" onClick={() => restoreVersion(csrftoken, formUuid, version.uuid, formAdminUrl)}>Herstellen</a></td>
```

Comment:
I think you need to capture the `event` here too and call `event.preventDefault()` otherwise the browser will jump to the top of the page because of the `#` href.